src/Scripting/CoreTest/ScriptOptionsTests.cs

```diff
@@ -148,5 +150,35 @@ public void WithImports_Errors()
             options.WithImports("b\0lah");
             options.WithImports(".blah");
         }
+
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
```

Comment:
nit: this is a bit of overkill.

Assert.True(ScriptOptions.Default.WithEmitDebugInformation(true));
Assert.False(ScriptOptions.Default.WithEmitDebugInformation(false));

would do.